### ISSUE TITLE
not ok 7 - flux-cron tab works for month

### ISSUE THREAD

**Issue by chu11** [flux-framework/flux-core#3976]
i think another variant of fixes in 

aeffda49a56ca94b7a8f56529ff37d983e80cdb9

```
lua: /usr/src/src/cmd/flux-cron:99: bad argument #2 to 'fmt' (number has no integer representation)
stack traceback:
	[C]: in function 'string.format'
	/usr/src/src/cmd/flux-cron:99: in upvalue 'reladate'
	/usr/src/src/cmd/flux-cron:497: in field 'handler'
	/usr/src/src/bindings/lua/flux/Subcommander.lua:133: in function </usr/src/src/bindings/lua/flux/Subcommander.lua:114>
	(...tail calls...)
	/usr/src/src/cmd/flux-cron:580: in main chunk
	[C]: in ?
```

line of issue

`return fmt ("%d weeks ago", (d + 3) / 7)`

I'll put a quick fix together

**Comment by chu11** [comment]
ends up @grondo already has some partial fixes in https://github.com/flux-framework/flux-core/pull/3844, just missed the one above line.  I'll assign this to @grondo so he can fixup whenever he gets back to PR #3844

**Comment by grondo** [comment]
Thanks @chu11!

**Comment by chu11** [comment]
i believe fixed in #3844
